AIMSICD/src/main/AndroidManifest.xml

```diff
@@ -26,6 +26,11 @@
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
 
     <!-- SPECIAL PERMISSIONS TO BE ADDED AFTER THIS LINE. /> -->
+    
+    <!-- Required: Used to deliver privacy-aware Bugsnag crash reports -->
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <!-- Optional to see which Activity was active at the time of a crash -->
+    <uses-permission android:name="android.permission.GET_TASKS"/>
```

Comment:
Ok, fair. Would you recommend me to remove these persmissions again, or keep them?